Description: Fix build on FreeBSD #650820
Author: Gabriele Giacone <1o5g4r8o@gmail.com>
Last-Update: 2012-07-24
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/lib/clplumbing/ipcsocket.c
+++ b/lib/clplumbing/ipcsocket.c
@@ -2406,7 +2406,7 @@ socket_verify_auth(struct IPC_CHANNEL* c
 
 #define	EXTRASPACE	0
 
-#ifdef HAVE_STRUCT_CMSGCRED
+#if defined(HAVE_STRUCT_CMSGCRED) || defined( __FreeBSD_kernel__) || defined(__GNU__)
 	/* FreeBSD */
   typedef struct cmsgcred Cred;
 #	define crRuid	cmcred_uid
